include/tkey/proto.h

```diff
@@ -38,6 +44,8 @@ uint8_t genhdr(uint8_t id, uint8_t endpoint, uint8_t status, enum cmdlen len);
 int parseframe(uint8_t b, struct frame_header *hdr);
 void writebyte(uint8_t b);
 void write(const uint8_t *buf, size_t nbytes);
```

Comment:
We add USB framing protocol support for the `read` and `readbyte` functions but not the `write` and `writebyte` functions. Is this intentional?